rpc/rpc-lib/src/rpc-clnt.c

```diff
@@ -1612,7 +1612,7 @@ rpc_clnt_submit(struct rpc_clnt *rpc, rpc_clnt_prog_t *prog, int procnum,
     };
     struct rpc_req *rpcreq = NULL;
     rpc_transport_req_t req;
-    int ret = -1;
+    int ret = -2;
```

Comment:
Ah, magic error numbers that no one knows what they stand for...

Reply:
It has been done in this way so as to handle the scenario of destroying the frame in case the first two checks fail in this function.

Reply:
But maybe we can support this with a comment so that somebody new wouldn't scratch their head :p